foursight_core/react/api/react_api.py

```diff
@@ -0,0 +1,447 @@
+                "es": app.core.host,
+                "rds": os.environ["RDS_HOSTNAME"],
+                # TODO: cache this (slow).
+                "sqs": app.core.sqs.get_sqs_queue().url,
+            },
+            # TODO: cache this (slow).
+            "buckets": {
+                "env": app.core.environment.get_env_bucket_name(),
+                "foursight": get_foursight_bucket(envname=env if env else default_env, stage=stage_name),
+                "foursight_prefix": get_foursight_bucket_prefix(),
+                "info": environment_and_bucket_info,
+                "ecosystem": sort_dictionary_by_lowercase_keys(EnvUtils.declared_data()),
+            },
+            "page": {
+                "path": request.get("context").get("path"),
+                "endpoint": request.get("path"),
+                "loaded": app.core.get_load_time()
+            },
+            # TODD: Move these out to another API.
+            "checks": {
+                "running": 0,
+                "queued": 0
+            },
+            "known_envs": self.envs.get_known_envs_with_gac_names(self.gac),
+            # TODO: cache this (slow).
+            "gac": {
+                "name": get_identity_name(),
+                "values": sort_dictionary_by_lowercase_keys(obfuscate_dict(get_identity_secrets())),
+             },
+            "environ": sort_dictionary_by_lowercase_keys(obfuscate_dict(dict(os.environ)))
+        }
+        return response
+
+    def reactapi_users(self, request: dict, env: str):
+        users = []
+        # TODO: Support paging.
+        user_records = ff_utils.get_metadata('users/', ff_env=full_env_name(env), add_on='frame=object&limit=10000')
+        for user_record in user_records["@graph"]:
+            last_modified = user_record.get("last_modified")
+            if last_modified:
+                last_modified = last_modified.get("date_modified")
+            # TODO
+            # roles = []
+            # project_roles = user_record.get("project_roles")
+            # if project_roles:
+            #     role = role.get("date_modified")
+            #     roles.append({
+            #         "groups": groups,
+            #         "project_roles": project_roles,
+            #         "principals_view": principals_view,
+            #         "principals_edit": principals_edit
+            #     })
+            users.append({
+                "email_address": user_record.get("email"),
+                "first_name": user_record.get("first_name"),
+                "last_name": user_record.get("last_name"),
+                "uuid": user_record.get("uuid"),
+                "modified": convert_utc_datetime_to_useastern_datetime_string(last_modified)})
+        response = self.create_standard_response("reactapi_users")
+        response.body = sorted(users, key=lambda key: key["email_address"])
+        return response
+
+    def reactapi_users_user(self, request: dict, env: str, email: str):
```

Comment:
This function name doesn't really immediately tell me what it does. If you're going to omit docstrings I would take special care that functions are named aptly so the code can be navigated reasonably.